Refactoring

TotallyNotShinobi opened the pull request:
added **text ID** for all articles

added **debug menu text**  to /EventText/
added **several skills** to /Skills/
added **Struggler's Prosthetic.md** to /Weapons/
added **Head of Ministry Rat.md** to /Items/
added **Interior Ministry Armor Memo.md** to /Items/
added **Father's Blessing_short.md** to /Items/

added **Red Guard Elite - Yamagata Masashige** to /NPCs/
added **Ashina Elite - Fujiwara Takawara** to /NPCs/
added **Bridgekeeper Shiramatsu** to /NPCs/
added **Wild Kenjustu Master** to /NPCs/

fixed several Japanese NPC names, item names, and descriptions
added Korean translation of **Struggler's Prosthetic** by ParkDongJin

TotallyNotShinobi committed:
added Skills, Weapons, and EventText

added Skills, Weapons, and EventText

TotallyNotShinobi committed:
added more NPC names and items

added more NPC names and items

PhantomKunai commented on Strings/EventText/EventText.md:
```diff
@@ -0,0 +1,44 @@
+|||
+
+# Aquire Bell Demon - Description
+leave it empty
```
Possible comment to be removed?

BaaridunNasr replied:
Possible comment to be removed?

BaaridunNasr commented on Strings/Items/Father's Blessing_short.md:
```diff
@@ -0,0 +1,13 @@
+|||
+| Japanese | <h3>**義父の加護**</h3>使えば、義父の加護をその身に受けられる<br><br>加護を受けているときに「霧がらす」を使用すると、<br>形代を2つ、追加で消費する代わりに、<br>短時間だけ義父のまぼろしと共闘できる<br><br>ただし、鬼仏で休息すると加護は消えてしまう<h3> |
+|||
+| Korean | <h3>**의부의 축복**</h3>Translation Needed<h3> |
```
What is the purpose of h3 tags here that is not used in Strings/EventText/EventText.md and other places

BaaridunNasr commented on Strings/NPCs/NPCs.md:
```diff
@@ -1,28 +1,82 @@
 | Italian | Translation needed |
 | Spanish | Translation needed |
 
+# Samurai General Ramiresu Arumando (Unused)
```
If its unused, might not make sense to add it unless its being put in game for sure. 
If so please let me know.

TotallyNotShinobi replied:
he is here because most of work here was also for my translation patch, which can be used for the older version of S:R (before Ramiresu removed)

BaaridunNasr replied:
> 
> 
> he is here because most of work here was also for my translation patch, which can be used for the older version of S:R (before Ramiresu removed)

@PrattiDev Can you comment if this can be removed in this patch if not needed? Maybe the translations can be kept as a record?

BaaridunNasr commented on Strings/NPCs/NPCs.md:
```diff
@@ -32,8 +86,34 @@
+| Korean | Translation needed |
+| German | Translation needed |
+| Italian | Translation needed |
+| Spanish | Translation needed |
```
Question : Should we put placeholders in other md files as well for Spanish language?

BaaridunNasr replied:
@TotallyNotShinobi 
If possible please add the placeholders for other languages only in the files you are currently sending for the PR

TotallyNotShinobi replied:
>  for other languages 

This means Korean, German, Italian, and Spanish, right?

BaaridunNasr commented on Strings/Skills/Murasama's Pestilence.md:
```diff
@@ -0,0 +1,15 @@
+| :------- | :---------- |
+| Text ID | 9063 |
+| Note | active but not in inventory<br>may get reworked in future |
+| | |
```
This spacing seems okay but would be good to maintain this everywhere. I dont see the same in [here](https://github.com/SekiroResurrection/mod-files/pull/2/files?short_path=cfb2e9d#diff-cfb2e9d53681ef69d6cf0b788a6af4d208779530c6e0ca024b63b3ddedaff23e)

@PrattiDev Whats your opinion?

BaaridunNasr replied:
To clarify by spacing I mean the extra rows between each row

TotallyNotShinobi replied:
I'd like to recommend using a spacing row only for Skills, Items, and Weapons which have several sentences per translations.
NPCs and EventTexts have one sentences each within several tables, so adding spacing between rows may reduces readability.

TotallyNotShinobi committed:
Update EventText.md

TotallyNotShinobi commented on Strings/Items/Crimson Tanto.md:
```diff
@@ -2,15 +2,16 @@
 | English | <h3>**Crimson Tanto**</h3>A Ceremonial Tanto of unknown origin. Costs Spirit Emblems to use.<br><br>Cut into one's own flesh to aquire Bestowal effect on the Sword.<br><br>This dagger is eerily similar to the Red Mortal Blade, thus it temporarily blocks off resurrective powers when used.<br><br>The blade is inscribed with its true name: "Devoted Flesh"<br><br>One needs to be careful w ...
 |||
-| Japanese | <h3>**血汐流し**</h3>出処の知れぬ真っ赤な短刀<br>ＨＰと形代を代償に、血を刀に纏わせる<br>この短刀は不気味なほどに、不死斬りと似通う<br>それゆえか、使えば一時的に回生の力は使えなくなる<br>刻まれた銘は「御贄」<br>それがこの短刀の真の名<br>力のために己を捨てる哀れな覚悟<br>それを持たねば、この短刀は扱えぬ<h3> |
+| Japanese | <h3>**血汐流し**</h3>出処の知れぬ深紅の短刀<br>ＨＰと形代を代償に、血を刀に纏わせる<br><br>この短刀は不気味なほどに、不死斬りと似通う<br>それゆえか、使えば一時的に回生の力は使えなくなる<br>力のために己を捨てる哀れな覚悟<br>それを持たねば、この短刀は扱えぬ<br><br>刻まれた銘は「御贄」<br>それがこの短刀の真の名<h3> |
```
【From Kiritanpo】
I prefer 「真っ赤な短刀」 over 「深紅の短刀」
the ceremonial tanto says 「真白い短刀」 so 真っ赤 sounds more jaxtaposed
likewise, I think「刻まれた銘は「御贄」<br>それがこの短刀の真の名」should be the last line, because it is also the case for the ceremonial tanto 
also very minor thing, but extra punctuation on the「出所の知れぬ、真っ赤な短刀」 makes it more Fromsoftware text-ish I think

TotallyNotShinobi commented on Strings/Items/Nightjar Firebomb.md:
```diff
@@ -2,11 +2,12 @@
 | English | <h3>**Nightjar Firebomb**</h3>A volatile explosive used by the Nightjar shinobi who defend Ashina.<br><br>Allows one to detonate a powerful explosive at the risk of death.<br><br>Nightjars are no strangers to the threat of death. In the face of adversity, it may even count as bravery, or perhaps it is reckless desperation.<br><br>Yet few can deny the results of this harbinger of death ...
 |||
-| Japanese | <h3>**寄鷹の焙烙玉**</h3>葦名の忍び、寄鷹衆が使う火薬玉<br>使えば、命と引き換えに巨大な爆発を引き起こす<br><br>寄鷹の者は、己の命さえ顧みぬ<br>それは勇猛さの顕れなのか、それともただの捨鉢か<br>炸裂の威力の前では、些細なことだ<h3> |
+| Japanese | <h3>**寄鷹の焙烙玉**</h3>葦名の忍び、寄鷹衆が使う火薬玉<br>使えば、体力と引き換えに巨大な爆発を引き起こす<br><br>寄鷹の者は、主命の為なら己の身さえ顧みぬ<br>それは勇猛さの顕れなのか、それともただの捨鉢か<br><br>炸裂の威力の前では、些細なことだ<h3> |
```
【From Kiritanpo】
I think we should add「時に」("sometime") before 「己の身さえ顧みぬ」
the text as of now gives the impression that nightjar is always reckless and does not regard their own lives all the time
by adding the word the reader can infer that nightjar CAN be reckless when they are needed to
I also suggest to change「炸裂の威力の前では、些細なことだ」to 「この炸裂の威力の前では、些細なことだ」
「炸裂」 means explosion, and by adding 「この」（"this") it specifies the explosion of the firebomb.

TotallyNotShinobi commented on Strings/Items/Head of Ministry Rat.md:
```diff
@@ -0,0 +1,13 @@
+|||
+| English | <h3>**Head of Ministry Rat**</h3>The head of an Interior Ministry Elite. <br><br>These elite soldiers and shinobis lead an invasion against the Ashina Castle. <br>Despite their high status and efforts, they are nothing more than just rats.<br><br>Can be traded with the Tengu for a reward. <br>(Not Implemented yet)<h3> |
+|||
+| Japanese | <h3>**鼠の首級**</h3>葦名の城に忍び込んだ、内府の間者の首<br><br>孤影の衆と行動を共にする赤備えは、<br>赤備えの中でも、選りすぐりの猛者である<br>だが、どれだけ精悍であろうと、所詮は鼠よ<br><br>天狗の元に持っていけば、<br>褒美と交換できるだろう<br> （未実装）<h3> |
```
【From Kiritanpo】
・Head of Ministry Rat
孤影の衆と行動を共にする赤備えは、赤備えの中でも、選りすぐりの猛者である
↓
孤影の衆と行動を共にする赤備えは、赤備えの中でも、葦名城攻めの先鋒を任された、選りすぐりの猛者たちである
（Reflecting on the original "These elite soldiers and shinobis lead an invasion against the Ashina Castle." English text）

TotallyNotShinobi commented on Strings/Items/Interior Ministry Armor Memo.md:
```diff
@@ -0,0 +1,13 @@
+|||
+| English | <h3>**Interior Ministry Armor Memo**</h3>A note detailing information on Interior Ministry plated armors.<br><br>The Kodachi-wielding enemies have a new special<br>type of armor, resistant to standard slashes.<br>The armor can be ripped off, leaving them<br>vulnerable to foreign substances and deep cuts.<br><br>The Tengu awaits the for heads of the generals with a reward.<h3> |
+|||
+| Japanese | <h3>**内府の鎧の覚書**</h3>内府の鎧について記された文<br><br>　鼠の鎧について記す<br><br>　・赤鎧を身に着け、小太刀を振るう<br>　・やつらの鎧は、南蛮由来の板金鎧<br>　・それゆえ、生中な斬撃は通じぬ<br>　・引き剥がせば、それまでのこと<br><br>褒美を求めるならば、鼠の首と共に天狗の元へ行くと良い<h3> |
```
【From Kiritanpo】
・ Interior Ministry Armor Memo
赤鎧を身に着け、小太刀を振るう　やつらの鎧は、南蛮由来の板金鎧　それゆえ、生中な斬撃は通じぬ　引き剥がせば、それまでのこと
↓
赤鎧を身に着け、小太刀を振るう曲者あり　やつらの鎧は、南蛮由来の板金鎧なり　それゆえ、生中な斬撃は通じぬ　しかし引き剥がしてしまえば、それまでのことだ

TotallyNotShinobi commented on Strings/Skills/Masamune's Edge.md:
```diff
@@ -0,0 +1,18 @@
+| | |
+| English | <h3>**Masamune's Edge**</h3><br>Masamune, an elite among the Interior Ministry ranks.<br>Labelled the "Crimson-Fang" for his red streaks among the battle field. <br><br>"The sprays of blood grew larger as the scarlet stimulants sank in."<br><br>Besting this shadow has bestowed the Wolf<br>with additional attack power after deathblows.<h3> |
+| | |
+| Japanese | <h3>**孤影の技・尖鋭**</h3><br>忍殺を成功させた後、数秒間<br>ＨＰへのダメージが上昇する常在効果<br><br>孤影の衆の正宗は、内府からの信頼も厚い<br>返り血を浴びつつ敵を仕留める様子から、<br>おのずと「紅牙」の名がついた<br><br>　天高く血飛沫が噴き上がるたび、<br>　血は滾り、血に酔えるのだ<h3>|
```
【From Kiritanpo】
・Masamune's edge
孤影の衆の正宗は、内府からの信頼も厚い
↓
孤影の衆が一人、正宗は内府が最も信頼する忍びの一人である

TotallyNotShinobi committed:
Update Crimson Tanto.md

TotallyNotShinobi reviewed (commented):
applied Kiritanpo's suggestion

TotallyNotShinobi committed:
Update Head of Ministry Rat.md

applied Kiritanpo's suggestion

TotallyNotShinobi committed:
Update Nightjar Firebomb.md

applied Kiritanpo's suggestion

TotallyNotShinobi committed:
Revert "Update Head of Ministry Rat.md"

This reverts commit 865584f6bfed1d0458f05788692003c34695e8bb.

TotallyNotShinobi committed:
spacing and languages

adjusted spacing and formatting for better readability
added other languages for future translation

TotallyNotShinobi committed:
Update Father's Blessing.md

TotallyNotShinobi committed:
Update Father's Blessing_short.md

BaaridunNasr reviewed (commented):
Nice work, @TotallyNotShinobi!

ED will take care of the final review and merge when he is available.

TotallyNotShinobi committed:
revised Japanese texts

revised Japanese texts with Kiritanpo's feedback

TotallyNotShinobi committed:
Update Head of Ministry Rat.md

fixed grammer
